plugins/adobe-analytics-compatible-version/index.ts

```diff
@@ -120,13 +121,19 @@ import { ValidationPluginResult } from 'types/validationPlugin';
     ? {
         events: [],
         message:
-          'The installed Assurance SDK and Analytics Edge Bridge Extensions are compatible.',
+          'The installed Assurance SDK and Edge Bridge extensions are compatible.',
         result: 'matched'
       }
+    : !analyticsVersion
+    ? {
+      message: 'The Analytics and Edge Bridge extensions are not installed, nothing to validate.',
+      events: [],
+      result: 'matched'
```

Comment:
the result should be `unknown` with a more generic message in case for scenarios when it's misconfiguration and the extension was mistakenly not registered.

Reply:
How about the new update message?